corehq/messaging/smsbackends/telerivet/migrations/0002_add_index_on_webhook_secret.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('telerivet', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='incomingrequest',
+            name='secret',
+            field=models.CharField(max_length=255, null=True, db_index=True),
```

Comment:
this migration should be quick, there aren't enough records yet on this table for it to be an issue